app/controllers/redeemify_codes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RedeemifyCodesController < ApplicationController
+
+  before_action :set_code, only: [:show]
+
+  def show
+    render json: @code
+  end
+
+  private
+
+  def set_code
+    if current_user then
```

Comment:
I'm going to be writing 'authenticate user' in the next ticket, so if it's OK with you I'll just leave as is for now.